bucket/meilisearch.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://github.com/meilisearch/meilisearch",
```

Comment:
This should be after `description` as per https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets